release_tooling/src/release.py

```diff
@@ -28,11 +28,19 @@
 @click.pass_context
 def main(ctx, aws_profile, project_file, verbose, dry_run):
     try:
-        project = project_config.load(project_file)
+        projects = project_config.load(project_file)
     except FileNotFoundError:
         message = f"Couldn't find project metadata file {project_file!r}.  Run `initialise`."
         raise click.UsageError(message) from None
 
+    if len(projects) == 1:
+        project_id = projects[0].id
+    else:
+        project_ids = [project['id'] for project in projects]
+        project_id = click.prompt(text="Enter the project ID", type=click.Choice(project_ids))
+
+    project = next(project for project in projects if project['id'] == project_id)
```

Comment:
This relies on projects not being assigned duplicate IDs (or the later project will never be accessible).

Do we have to store projects as an array, or could we store them in a map, keyed by project ID? i.e.

```json
{
  "id1": { ... project 1 ... },
  "id2": { ... project 2 ... },
  ...
}
```

Reply:
I approve this message.